tests/unit_tests/scheduler/test_lsf_driver.py

```diff
@@ -130,6 +130,15 @@ async def test_submit_with_default_queue():
     assert "-q" not in Path("captured_bsub_args").read_text(encoding="utf-8")
 
 
+@pytest.mark.usefixtures("capturing_bsub")
+async def test_submit_with_resource_request():
```

Comment:
I think we can afford an integration test toward the real cluster also on this. Just use some low requirements that seems relevant and write test function similiar to test_submit in test_generic_driver that writes the string test to a file, and then assert that the string can be found.